app/views/categories/_concept.html.erb

```diff
@@ -1,4 +1,4 @@
 <div class="<%= wrap_class %>">
   <%= link_to concept.name, concept_path(concept), class: "govuk-body strong_link" %>
-  <p class="govuk-!-font-size-16 govuk-!-margin-top-1"><%= concept.description %></p>
+  <p class="govuk-!-font-size-16 govuk-!-margin-top-1"><%= concept.description || concept.placeholder_description %></p>
```

Comment:
This looks good :+1:

As it's a potentially expensive action (particularly where we have many `data_element`s, do we need partial caching here? I'd keep that separate from this PR anyway, so it doesn't block merging...